ote_sdk/ote_sdk/entities/model.py

```diff
@@ -404,6 +406,16 @@ def model_size_reduction(self) -> float:
     def model_size_reduction(self, value: float):
         self.__model_size_reduction = value
 
+    @property
+    def exportable_code(self) -> bytes:
+        """
+        Get the exportable_code
+        """
+        return self.__exportable_code
+
+    def set_deployable_model(self, data: bytes):
```

Comment:
Not sure about the proposed design, but it is much more natural to define it a property setter:

```suggestion
    @exportable_code.setter
    def exportable_code(self, data: bytes):
```